packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+import { TMarketTypes, TPlatforms } from './types';
+import FinancialMT5Icon from '../../public/images/mt5-financial.svg';
+import SwapFreeMT5Icon from '../../public/images/mt5-swap-free.svg';
+import DerivedMT5Icon from '../../public/images/mt5-derived.svg';
+
+export const MarketTypeToTitleMapper: Record<TMarketTypes.All, string> = {
+    all: 'Swap-Free',
```

Comment:
this seems like translations responsibility - would it be possible to squeeze that into translations / localisations? ideally we shouldn't have any translations hardcoded in javascript,